server/lib/defaults.js

```diff
@@ -29,12 +29,6 @@ exports.test = {
 };
 
 exports.testPageContext = {
-  home: true,
```

Comment:
decided to separate these template logic vars from the test defaults. not DRY but easier to reason about